token/js/src/extensions/transferHook/instructions.ts

```diff
@@ -190,14 +192,120 @@ export async function addExtraAccountsToInstruction(
     return new TransactionInstruction({ keys: accountMetas, programId, data: instruction.data });
 }
 
+/**
+ * Construct an `ExecuteInstruction` for a transfer hook program
```

Comment:
nit: just to make it clear what this function is meant for:
```suggestion
 * Construct an `ExecuteInstruction` for a transfer hook program, without the additional accounts
```